dashboard/static/js/Roles.js

```diff
@@ -30,15 +30,75 @@ $(document).ready(function(){
         }
     });
     
+    function loadRolesTable(table, tableRows, columnAmt) {
```

Comment:
There's already a function that is supposed to handle creating a table inside the table.js file, so remaking this here is unnecessary. Instead, the code in table.js should be refactored to work here and on the employee page to remove redundant code.